tests/func/test_version.py

```diff
@@ -0,0 +1,45 @@
+import re
+
+from dvc.main import main
+
+
+def test_run_in_repo(dvc_repo):
```

Comment:
There is no need in `test_run_in_repo` and `test_run_outside_of_repo`, since you are testing them in `test_info_in_repo` and `test_info_outside_of_repo`.